com.unity.cinemachine/Runtime/Behaviours/CinemachineFreeLook.cs

```diff
@@ -549,7 +549,7 @@ private CinemachineVirtualCamera[] CreateRigs(CinemachineVirtualCamera[] copyFro
         {
             // Invalidate the cache
             mOrbitals = null;
-            float[] softCenterDefaultsV = new float[] { 0.5f, 0.55f, 0.6f };
+            float[] softCenterDefaultsV = new float[] { 0.5f, 0.5f, 0.5f };
```

Comment:
why this change?

Reply:
Ah, it was for my tests. So that the default freelook I create is upgradable. It should be reverted.

Reply:
It's upgradable as-is, with FreeLookModifier (use Tilt modifier)

Reply:
How should I convert? It's a different approach, and the conversion seems complex.
RotateToScreenBounds